README.md

```diff
@@ -232,13 +248,14 @@ Please make sure to include `#gene_id` in the first line. The default exclusion
 
 There are 7 different output files produced by `staramr`:
```

Comment:
Could you change **7 different output files** to **8 different output files**?

Reply:
Hey Aaron, the images/settings_example.png image is under https://github.com/phac-nml/staramr/tree/update_docs_0.6.0#settingstxt where the example section is thats where the updated info is located.

Also I've updated the number from 7 to 8.

Reply:
Woops. Thanks. I must have missed that image.